themes/socialbase/templates/block/block--social-follow-content-filter.html.twig

```diff
@@ -0,0 +1 @@
+{% extends "block--views-exposed-filter-block.html.twig" %}
```

Comment:
The fact that this works shows that we've tried to rebuild a view without views.

Reply:
Do you propose paste code in this template from template **block--views-exposed-filter-block.html.twig** without any changes?